keel/Core/storage_backends.py

```diff
@@ -14,3 +15,8 @@ class PrivateMediaStorage(S3Boto3Storage):
     file_overwrite = False
     custom_domain = False
 
+doc_storage = None
+if settings.DEBUG == True:
+	doc_storage = default_storage
+else:
+	doc_storage = PrivateMediaStorage()
```

Comment:
will we be categorising this for every model that uses Private media storage
Lets manage the PrivateMediaStorage class only to use Stoage locations that are already managed for different Environments

Reply:
No. We will not need to categorize for any further models. 

If any model needs to use this, they need to set 
storage = default_storage. Based on DEBUG, it will auto save to files to local or S3 bucket.

Reply:
this would be an extra step then if we implement any file field 
but if we have the storage backend there for us...
why not let it only handle where to store depending on the env